packages/state-transition/src/util/execution.ts

```diff
@@ -63,7 +63,6 @@ export function isMergeTransitionBlock(state: BeaconStateExecutions, body: bella
  * Merge is complete when the state includes execution layer data:
  * state.latestExecutionPayloadHeader NOT EMPTY
  */
-// TODO GLOAS: See if we need to update this for gloas
```

Comment:
we should just remove the merge transition code, it's probably simpler than making it somehow work with gloas